tmt/steps/provision/beaker_mrack.py

```diff
@@ -0,0 +1,532 @@
+import dataclasses
+import datetime
+import sys
+from typing import Any, Dict, List, Optional, Tuple, cast
+
+import click
+
+import tmt
+import tmt.options
+import tmt.steps
+import tmt.steps.provision
+import tmt.utils
+from tmt.utils import ProvisionError, updatable_message
+
+if sys.version_info >= (3, 8):
+    from typing import TypedDict
+else:
+    from typing_extensions import TypedDict
+
+from asyncio import get_event_loop
+from asyncio.coroutines import coroutine  # type: ignore
+from functools import update_wrapper
+
+from mrack.providers import providers
+from mrack.providers.beaker import PROVISIONER_KEY as BEAKER
+from mrack.providers.beaker import BeakerProvider
+from mrack.transformers.beaker import BeakerTransformer
+
+import mrack
+
+
+def async_run(func: Any) -> Any:
+    """Decorate click actions to run as async."""
+    func = coroutine(func)
+
+    def wrapper(*args: Dict[Any, Any], **kwargs: Dict[Any, Any]) -> Any:
```

Comment:
`*Any` and `**Any` are the correct types for `*args` and `**kwargs`.

Reply:
For some reason it reports invalid syntax I am sorry if i misunderstood.

Reply:
Sorry, mea culpa: `*args: Any, **kwargs: Any` is indeed the correct way.